app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (NotWearingMaskError,
+                        NotVaccinatedError,
+                        OutdatedVaccineError)
+
+
+def go_to_cafe(friends, cafe) -> str:
+    total = 0
```

Comment:
perhaps a bad name for variable - it is too general and need explains
```suggestion
    masks_needed = 0
```